skeleton/Rakefile

```diff
@@ -1,5 +1,6 @@
 require 'puppetlabs_spec_helper/rake_tasks'
-require 'puppet/vendor/semantic/lib/semantic'
+require 'puppet/version'
+require 'puppet/vendor/semantic/lib/semantic' unless Puppet.version.to_f < 3.6
```

Comment:
phew! :+1: 
I thought I was missing something. Thanks for keeping me on my toes though! :godmode: